backend/i18n/source/backend_it.json

```diff
@@ -23,24 +23,6 @@
   "apps.roles.nameAlreadyExists": "Esiste già un ruolo con lo stesso nome.",
   "apps.roles.usedRoleByClientsNotRemovable": "Non è possibile rimuovere un ruolo quando è assegnato ad un ruolo.",
   "apps.roles.usedRoleByContributorsNotRemovable": "Non è possibile rimuovere un ruolo quando questo è assegnato ad un collaboratore.",
-  "aspnet_annotations_AbsoluteUrl": "Il campo {0} deve essere un URL assoluto.",
```

Comment:
Sorry, I only meant that you should remove the aspnet_annotations_XXX translations, NOT `aspnet_identity_XXX`

Reply:
I have removed all labels that weren't in the origin file

Reply:
Please add the missing label

Reply:
You just have to get the old translations back and rename the keys, see: https://docs.squidex.io/01-getting-started/contributing-and-developing/translating#how-to-fix-errors